scripts/mergeITs.js

```diff
@@ -111,13 +111,21 @@ async function createPom() {
 // copy a file
 function copyFileSync(source, target, replaceCall) {
   var targetFile = target;
-  //if target is a directory a new file with the same name will be created
+  // if target is a directory a new file with the same name will be created
   if (fs.existsSync(target)) {
     if (fs.lstatSync(target).isDirectory()) {
       targetFile = path.join(target, path.basename(source));
     }
   }
   if (fs.existsSync(targetFile)) {
+    // When file exists we can merge both or override depending on the type
+    if (/.properties$/.test(source)) {
+      let content = fs.readFileSync(source, 'utf8');
+      content += fs.readFileSync(targetFile, 'utf8');
```

Comment:
This kind of relies on having a trailing newline in the first file, which isn't enforced. Maybe we should add a newline between both file contents?

Reply:
done